src/components/utilities/Catcher.test.tsx

```diff
@@ -28,7 +28,7 @@ describe('Catcher', () => {
 	});
 
 	test('Render a component with an error', () => {
-		jest.spyOn(console, 'error');
+		jest.spyOn(console, 'error').mockImplementation();
```

Comment:
What is the purpose of the empty mocked implementation?

Reply:
Avoid test fails after jest-fail-on-console dependency introduction